test/src/main/java/io/zeebe/test/exporter/ExporterIntegrationRule.java

```diff
@@ -0,0 +1,327 @@
+   *
+   * @param visitor record consumer
+   */
+  public void visitExportedRecords(Consumer<Record<?>> visitor) {
+    RecordingExporter.getRecords().forEach(visitor);
+  }
+
+  /**
+   * Deploys the given workflow to the broker. Note that the filename must have the "bpmn" file
+   * extension, e.g. "resource.bpmn".
+   *
+   * @param workflow workflow to deploy
+   * @param filename resource name, e.g. "workflow.bpmn"
+   */
+  public void deployWorkflow(BpmnModelInstance workflow, String filename) {
+    clientRule.getClient().newDeployCommand().addWorkflowModel(workflow, filename).send().join();
+  }
+
+  /**
+   * Creates a workflow instance for the given process ID, with the given payload.
+   *
+   * @param processId BPMN process ID
+   * @param payload initial payload for the instance
+   * @return unique ID used to interact with the instance
+   */
+  public long createWorkflowInstance(String processId, Map<String, Object> payload) {
+    return clientRule
+        .getClient()
+        .newCreateInstanceCommand()
+        .bpmnProcessId(processId)
+        .latestVersion()
+        .payload(payload)
+        .send()
+        .join()
+        .getWorkflowInstanceKey();
+  }
+
+  /**
+   * Creates a new job worker that will handle jobs of type {@param type}.
+   *
+   * <p>Make sure to close the returned job worker.
+   *
+   * @param type type of the jobs to handle
+   * @param handler handler
+   * @return a new JobWorker
+   */
+  public JobWorker createJobWorker(String type, JobHandler handler) {
+    return clientRule.getClient().newWorker().jobType(type).handler(handler).open();
+  }
+
+  /**
+   * Publishes a new message to the broker.
+   *
+   * @param messageName name of the message
+   * @param correlationKey correlation key
+   */
+  public void publishMessage(String messageName, String correlationKey) {
+    clientRule
+        .getClient()
+        .newPublishMessageCommand()
+        .messageName(messageName)
+        .correlationKey(correlationKey)
+        .send()
+        .join();
+  }
+
+  /**
+   * Blocks and wait until the workflow identified by the key has been completed.
+   *
+   * @param workflowInstanceKey ID of the workflow
+   */
+  public void awaitWorkflowCompletion(long workflowInstanceKey) {
+    TestUtil.waitUntil(
+        () ->
+            workflowInstanceRecords(WorkflowInstanceIntent.ELEMENT_COMPLETED)
+                .filter(r -> r.getKey() == workflowInstanceKey)
+                .exists());
+  }
+
+  private Properties newClientProperties() {
+    final Properties properties = new Properties();
+    properties.put(
+        ClientProperties.BROKER_CONTACTPOINT,
+        getBrokerCfg().getGateway().getNetwork().toSocketAddress().toString());
+
+    return properties;
+  }
+
+  private ExporterIntegrationRule configure(List<ExporterCfg> exporters) {
+    getBrokerCfg().getExporters().addAll(exporters);
+
+    return this;
+  }
+
+  @FunctionalInterface
+  public interface RecordVisitor {
```

Comment:
I think also this comment got lost, what is the point of this interface? The visit https://github.com/zeebe-io/zeebe/pull/1819/files#diff-71080026c1543e982625973ae06c0c0dR231 method uses a consumer as parameter.

Reply:
Also looks like a left over from a previous iteration :see_no_evil: